Power Up Proto/src/org/usfirst/frc/team5493/robot/subsystems/OneClimbyBoi.java

```diff
@@ -0,0 +1,70 @@
+package org.usfirst.frc.team5493.robot.subsystems;
+
+import org.usfirst.frc.team5493.robot.Robot;
+import org.usfirst.frc.team5493.robot.RobotMap;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+import edu.wpi.first.wpilibj.CounterBase.EncodingType;
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ *
+ */
+public class OneClimbyBoi extends Subsystem {
+
+	public WPI_TalonSRX climbMotor;
+	public Object climbDown;
+	
+	public static final double kDistancePerRevolution = 18.84;
+	public static final double kPulsesPerRevolution = 1024;
+	public static final double kDistancePerPulse = kDistancePerRevolution / kPulsesPerRevolution;
+	
+	//Encoder numbers to be added
+	//Encoder A Channel -> DIO 1, Module 1;
+	
+	//private Encoder encoder = new Encoder(1, 2, false, EncodingType.k2X);
+	
+	
+	public void robotInit() {
+		//encoder.setDistancePerPulse(kDistancePerPulse);
+		
+		//encoder.start();
+		
+		resetEncoders();
+	}
+	
+	public OneClimbyBoi() {
+	super();
+	
+	climbMotor = new WPI_TalonSRX(RobotMap.CLIMBER);
+    
+	 }
+    public void initDefaultCommand() {
+        
+    }
+    public void climbUp() {
+    	Robot.climber.climbMotor.set(0.8);
+    }
+    public void climbDown(){
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Rename method "climbDown" to prevent any misunderstanding/clash with field "climbDown" defined on line 18. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1845)